StripeConnect/StripeConnect/StripeConnect.docc/StripeConnect.md

```diff
@@ -0,0 +1 @@
+# ``StripeConnect``
```

Comment:
@porter-stripe Do we now use `.docc` files in the SDKs? I'm guessing this might have been auto-generated by Xcode and we might want to delete it since we do our own docs generation.

Reply:
I see that PaymentSheet has an empty placeholder for its `.docc` directory:
https://github.com/stripe/stripe-ios/blob/e9b3c5bb/StripePaymentSheet/StripePaymentSheet/Docs.docc/StripePaymentSheet.md

If Xcode needs this for something, we can probably keep it (maybe rename to `Docs.docc` it to be consistent with the other modules?) Otherwise I think we should remove it.

Reply:
I removed it. I checked other SDKs and it looks like any that have the `.docc` have no content in them.

Reply:
Sounds good, yeah they are auto generated